qt/scientific_interfaces/Inelastic/Analysis/FunctionBrowser/ConvFunctionModel.cpp

```diff
@@ -247,14 +272,27 @@ void ConvFunctionModel::removeBackground() {
 bool ConvFunctionModel::hasBackground() const { return m_backgroundType != BackgroundType::None; }
 
 EstimationDataSelector ConvFunctionModel::getEstimationDataSelector() const {
-  return [](const Mantid::MantidVec &, const Mantid::MantidVec &,
-            const std::pair<double, double>) -> DataForParameterEstimation { return DataForParameterEstimation{}; };
+  return [](const Mantid::MantidVec &x, const Mantid::MantidVec &y,
+            const std::pair<double, double> &range) -> DataForParameterEstimation {
+    (void)range;
+
+    auto const maxElement = std::max_element(y.cbegin(), y.cend());
+    if (maxElement == y.cend())
+      return DataForParameterEstimation{{}, {}};
+
+    auto const maxElementIndex = std::distance(y.cbegin(), maxElement);
```

Comment:
You might also want to consider subtracting the minimum non-zero y-value as an estimate for the background - this may not be very reasonable, it depends on the form of the background of course, but it's probably better than nothing! Don't mind though if you want to get this merged as it seems to be a big improvement.

Reply:
Not suitable. A wide Lorentzian does not go to zero at large energies and can be confused with background

Reply:
Fair enough, thanks for clarifying @SpencerHowells